cmsis/device/rtos/mbed_lib.json

```diff
@@ -4,7 +4,7 @@
         "present": 1,
          "main-thread-stack-size": {
             "help": "The size of the main thread's stack",
-            "value": 4096
```

Comment:
This should be separate commit but the main thread should be changed in the application, this would not fit smaller devices.

Reply:
Got it, I will move to the appropriate place.